docs/model-serving/quickstart.md

```diff
@@ -0,0 +1,175 @@
+    ```
+
+To learn more about managing models with the Model Registry see the [Model Registry documentation](../model-registry/quickstart.md).
+
+
+### Deployment
+
+Assuming you have already created a model in the Model Registry, a deployment can now be created to prepare a model artifact for this model and make it accessible for running predictions behind a REST endpoint. A model artifact is a package containing all necessary files for the deployment of a model, including the model file and/or custom scripts for loading the model (a.k.a. predictor script) or transforming the model inputs at inference time (a.k.a. transformer script).
+
+See the [example notebooks](https://github.com/logicalclocks/hops-examples/blob/master/notebooks/ml).
+
+#### Predictors and Transformers
+
+Predictors are in charge of running a model server that loads a trained model, handles inference requests and returns predictions (see the [Predictor Guide](../generated/model-serving/predictor.md)). These inference requests can be transformed at inference time by configuring a transformer.
+
+Transformers are used to apply transformations on the model inputs before sending them to the model for making predictions (see the [Transformer Guide](../generated/model-serving/transformer.md)).
+
+As an example, consider a model trained to detect unusual activity of users within a platform. Instead of sending all the recent activity records of a user each time we want to predict if the use is abnormal, we can simply send the _user_id_ and build/enrich the model input in the transformer before sending it to the predictor, by retrieving the most recent activity from an external source such as a [Feature Store](https://docs.hopsworks.ai/feature-store-api/latest/).
+
+#### Create
+
+Apart from the `model.deploy()` shortcut available in the model metadata object, deployments can be created in two additional ways: deploying a predictor or creating a deployment from scratch.
+
+To deploy a predictor, the Model Serving handle provides the `ms.create_predictor()` method to define the predictor metadata object which can be deployed using the `predictor.deploy()` method.
+
+=== "Python"
+
+    ```python
+    # Define a predictor for a trained model
+    predictor = ms.create_predictor(model)
+
+    # Deploy the predictor
+    deployment = predictor.deploy()
+    ```
+
+Alternatively, deployments can be created and saved using the `ms.create_deployment()` and `deployment.save()` methods, respectively.
+
+=== "Python"
+
+    ```python
+    # Define a predictor for a trained model
+    predictor = ms.create_predictor(model)
+
+    # Define a deployment
+    deployment = ms.create_deployment(predictor)
+
+    # Save the deployment in the Model Serving
+    deployment.save()
+    ```
+
+By default, the name of the deployment is inferred from the model name. To learn about the different parameters to configure a deployment see the [Deployment Guide](../generated/model-serving/deployment.md).
```

Comment:
the name of the deployment is inferred from the model name ->
the name of the deployment is the same as the model name

Reply:
It's not exactly the same. We have to remove dashes, underscores for example.